src/compiler.jl

```diff
@@ -9416,18 +9482,18 @@ end
 # JIT
 ##
 
-function _link(job, (mod, adjoint_name, primal_name, ctx, TapeType))
+function _link(job, (mod, adjoint_name, primal_name, TapeType))
     # if job.config.params.ABI <: InlineABI
     #     return CompileResult(Val((Symbol(mod), Symbol(adjoint_name))), Val((Symbol(mod), Symbol(primal_name))), TapeType)
     # end
 
     # Now invoke the JIT
     jitted_mod = JIT.add!(mod)
-    if VERSION >= v"1.9.0-DEV.115"
-        LLVM.dispose(ctx)
-    else
-        # we cannot dispose of the global unique context
-    end
+    #if VERSION >= v"1.9.0-DEV.115"
+    #    LLVM.dispose(ctx)
```

Comment:
I think we still need to dispose this, cc @vchuravy

Reply:
if this matches the Context() in enzyme_call then it's now disposed on exit of enzyme_call